### DISCUSSION TITLE
Question about how to use

### DISCUSSION THREAD

**Discussion by YZXyinzhaoxu** [jensengroup/propka#174] [Q&A]
Excuse me, I was wondering where “python-m propka” will be used after downloading the propka python module? Use it in python/ jupyter notebook, or in windows power shell？

**Comment by sobolevnrm** [comment]
You can use it as a standalone executable or module.  Please see <https://propka.readthedocs.io/en/latest/quickstart.html> for more information.

**Reply by YZXyinzhaoxu** [reply]
sorry, but I can't find the location of the output file after using propka3 FILENAME command

**Reply by sobolevnrm** [reply]
This is insufficient information for us to help you.
